moebot_bot/bot/commands/random.go

```diff
@@ -0,0 +1,34 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/camd67/moebot/moebot_bot/bot/permissions"
+	"github.com/camd67/moebot/moebot_bot/util/db"
+	"github.com/camd67/moebot/moebot_bot/util/reddit"
+)
+
+type RandomCommand struct {
+	PermChecker  permissions.PermissionChecker
+	RedditHandle *reddit.Handle
+}
+
+func (ac *RandomCommand) Execute(pack *CommPackage) {
+	send, err := ac.RedditHandle.GetRandomImage("awwnime")
+	if err != nil {
+		pack.session.ChannelMessageSend(pack.channel.ID, "Ooops... Looks like this command isn't working right now. Sorry!")
+		return
+	}
+
+	pack.session.ChannelMessageSendComplex(pack.channel.ID, send)
+}
+
+func (ac *RandomCommand) GetPermLevel() db.Permission {
+	return db.PermAll
+}
+func (ac *RandomCommand) GetCommandKeys() []string {
+	return []string{"RANDOM", "R"}
+}
+func (ac *RandomCommand) GetCommandHelp(commPrefix string) string {
+	return fmt.Sprintf("`%[1]s r` or `%[1]s random` - Posts a cute anime character.", commPrefix)
```

Comment:
Since we haven't had a command yet that has multiple command keys we haven't dealt with this, but let's just keep this to `%[1]s random` instead of listing off both. Ideally this would show up in extended help (yet to be implemented)